drivers/spi/Kconfig.dw

```diff
@@ -20,4 +20,11 @@ config SPI_DW_ACCESS_WORD_ONLY
 	  DesignWare SPI only allows word access, byte access will raise
 	  exception.
 
+config SPI_DW_HSSI
+	bool "Designware SPI HSSI variant"
+	default y if $(dt_has_compat,$(DT_COMPAT_NORDIC_NRF_EXMIF))
```

Comment:
Remove that line. Such dt compat stuff is meant for HAL only afaik. 
The board that support such variant of registers will only need to set this config option in their defconfig and that's it.

Reply:
Removed